src/components/Calculator.js

```diff
@@ -0,0 +1,34 @@
+import './Calculator.scss';
+
+function Calculator() {
+  return (
+    <article className="calculator-container">
+      <div className="screen">0</div>
+      <div className="operator">
+        <div className="button">/</div>
```

Comment:
- [ ] This symbol should be &div;, not /, based on the design. If you do not have the sign on your keyboard consider using HTML entity code for it.